test/smartseq2_single_sample/pr/dependencies.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ss2_single_sample.wdl": "/working/smartseq2_single_sample/ss2_single_sample.wdl",
+  "ValidateSmartSeq2SingleCell.wdl": "/working/test/smartseq2_single_sample/pr/ValidateSmartSeq2SingleCell.wdl",
+  "hisat2_QC_pipeline.wdl": "/working/smartseq2_single_sample/pipelines/hisat2_QC_pipeline.wdl",
+  "hisat2_rsem_pipeline.wdl": "/working/smartseq2_single_sample/pipelines/hisat2_rsem_pipeline.wdl",
+  "hisat2.wdl": "/working/pipelines/tasks/hisat2.wdl",
+  "picard.wdl": "/working/pipelines/tasks/picard.wdl",
+  "rsem.wdl": "/working/pipelines/tasks/rsem.wdl"
+}
```

Comment:
What's going on with these paths? Does the repo get cloned into `/working` when you run the tests in jenkins?

Reply:
Correct.

Reply:
Working is also set as `cwd`.

Reply:
Thanks for clarifying. I think this is probably letting the details of how jenkins is executing the test bleed into the definition of the test. But we can address it when we reorganize the repo.

Reply:
Sounds good...  I think the long term goal is to run the portability service in lieu of these tests, but would be interested to hear your thoughts during the reorg discussion. :+1: